syntax = "proto3"; // PB协议版本
package aiins;

option java_package = "com.example.aiins.proto";

// 登录请求
message RegisterReq {
  string username = 1;
  string nickname = 2;
  string password = 3;
}

// 登录响应
message RegisterRsp {
  int32 resultCode = 1;
  int32 uid = 2;
}